document/core/valid/instructions.rst

```diff
@@ -1424,7 +1424,7 @@ Control Instructions
   * For all :math:`l_i` in :math:`l^\ast`,
     and for each :ref:`operand type <syntax-opdtype>` :math:`t_j` in :math:`t^\ast` and corresponding type :math:`t'_{ij}` in :math:`C.\CLABELS[l_i]`, :math:`t_j` :ref:`matches <match-opdtype>` :math:`t'_{ij}`.
 
-* Then the instruction is valid with type :math:`[t_1^\ast~t^\ast~\I32] \to [t_2^\ast]`, for any sequences of :ref:`value types <syntax-valtype>` :math:`t_1^\ast` and :math:`t_2^\ast`.
+* Then the instruction is valid with type :math:`[t_1^\ast~t^\ast~\I32] \to [t_2^\ast]`, for any sequences of :ref:`operand types <syntax-opdtype>` :math:`t_1^\ast` and :math:`t_2^\ast`.
```

Comment:
I cannot comment on line 1420, but it states "result type". I believe it should be the equivalent using operand types. The bullet points below correctly mention operand types.

Reply:
Right, fixed!